setup.py

```diff
@@ -53,7 +53,14 @@ def get_require_version(name):
         requires.append(get_require_version('%s_%s' % (prefix, dep)))
 requires.append(get_require_version('trytond'))
 
-tests_require = []
+extras_require = {
+    'dev': [
+        get_require_version('trytond-factories'),
+    ],
+}
+
+tests_require = extras_require['dev']
```

Comment:
This variable is unused, but actually setuptools deprecated the tests_require option so better remove altogether and stick with the tests|dev extras_require